README.md

```diff
@@ -65,25 +78,16 @@ Create the child application (see [docs](/docs/child-applications.md) for more d
 document.getElementById('main-content').textContent += "App1 is loaded.";
 
 // apps/app1.js
-export function bootstrap() {
-  return new Promise((resolve, reject) => {
-    document.getElementById('main-content').textContent += "App1 is bootstrapped.";
-    resolve();
-  });
+export async function bootstrap() {
+  document.getElementById('main-content').textContent += "App1 is bootstrapped.";
 }
 
-export function mount() {
-  return new Promise((resolve, reject) => {
-    document.getElementById('main-content').textContent += "App1 is mounted!";
-    resolve();
-  });
+export async function mount() {
+  document.getElementById('main-content').textContent += "App1 is mounted!";
 }
 
-export function unmount() {
-  return new Promise((resolve, reject) => {
-    document.getElementById('main-content').textContent = "";
-    resolve();
-  });
+export async function unmount() {
+  document.getElementById('main-content').textContent = "";
 }
```

Comment:
Yeah that's a good point. I use async functions in some places and just normal promises in other places so that people can see both ways. But probably this main Readme would be better with just normal promises, since this is supposed to be a "simple" getting started example.

Will change.